blockchain/blockchain-wallet-v4-frontend
# Feature/PL-bug-fixes

## Description
Misc bug fixes: 644, 284, 679, 703, 338, 655

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
644: Send bch to external address
284: Send from btc/bch/eth with second password
679: Toggle fiat/crypto display, see dropdown currency updates
703: Send btc/bch/eth from safari

## PR Creator Checklist

## PR Reviewer Checklist
